scripts/variantstore/wdl/GvsPrepareRangesCallset.wdl

```diff
@@ -114,7 +114,7 @@ task PrepareRangesCallsetTask {
   }
 
   runtime {
-    docker: "us.gcr.io/broad-dsde-methods/variantstore:2023-06-01-alpine"
+    docker: "us.gcr.io/broad-dsde-methods/variantstore:2023-06-05-alpine-bc1f7a5a2"
```

Comment:
now that we are going to updating these multiple times a pr---would it make sense to have a constant somewhere?

Reply:
I don't understand what you mean, happy to discuss internally